middleware/passport.js

```diff
@@ -77,6 +77,10 @@ module.exports = function (passport, app) {
               if (user) {
                 return done(null, false)
               } else {
+                if (!req.body.name) {
+                  return done('Name is required')
```

Comment:
Question @ecfy what happens if you dont pass email/password?

Reply:
Passport takes care of returning a 400 with "Bad Request" for those cases